packages/visual-editor/src/store/entityStore.ts

```diff
@@ -21,7 +21,7 @@ export const useEntityStore = create<EntityState>((set) => ({
   },
   resetEntityStore(locale, entities = []) {
     console.debug(
-      `[exp-builder.sdk] Resetting entity store because the locale changed to '${locale}'.`
+      `[exp-builder.sdk] Resetting entity store because the locale changed to '${locale}'.`,
```

Comment:
Should all the log messages that contain `exp-builder.sdk` and `exp-builder.visual-editor` be updated too?